### PR TITLE
:globe_with_meridians: Normalize timestamp. Comments and nmap verified.

### LINKED ISSUE #26: normalize timestamps
nmap and comments are utc. translate to a time zone set in the config for the frontend

### PR DESCRIPTION
**Pull request by ianalin123**
Resolves #26 

Changes:
- Added `timezone` field in `app.conf`
- Defined `normalizeTime` function in `comments.go`
- Used in `comments.go` and `nmap.go`

### CODE REVIEW

File: conf/app.conf [hunk 1]
```diff
@@ -1,6 +1,7 @@
 appname = raven
 runmode = dev
 httpport = 8080
+timezone = "Asia/Shanghai"
```
Comment (nasirhm) [inline conf/app.conf:4]: @ianalin123 set default to "America/Los_Angeles"

File: models/comments.go [hunk 2]
```diff
@@ -18,11 +19,19 @@ func init() {
 	orm.RegisterModel(new(Comment))
 }
 
-func FormatTime(t time.Time) string {
-    // Format the time according to the desired format
-    formattedTime := t.Format("Jan. _2, 2006 03:04PM")
+func normalizeTime(t time.Time) time.Time {
+
```
Comment (nasirhm) [inline models/comments.go:23]: Additional space.

**Review by nasirhm — CHANGES_REQUESTED** [review]

File: models/comments.go (hunk 2 above)
Comment (soh3ee) [reply models/comments.go:23]: > Additional space.

Can you elaborate lol

Comment (ianalin123) [reply models/comments.go:23]: i think he means the extra line of empty space lollll

Comment (nasirhm) [reply models/comments.go:23]: `go fmt` should resolve these issues.

**Review by nasirhm — APPROVED** [review]
LGTM

**Pull request merged**